InnerEye/ML/Histopathology/models/deepmil.py

```diff
@@ -148,21 +159,29 @@ def get_metrics(self) -> nn.ModuleDict:
         if self.n_classes > 1:
             return nn.ModuleDict({'accuracy': Accuracy(num_classes=self.n_classes, average='micro'),
                                   'macro_accuracy': Accuracy(num_classes=self.n_classes, average='macro'),
-                                  'weighted_accuracy': Accuracy(num_classes=self.n_classes, average='weighted')})
+                                  'weighted_accuracy': Accuracy(num_classes=self.n_classes, average='weighted'),
+                                  'confusion_matrix': ConfusionMatrix(num_classes=self.n_classes)})
         else:
             return nn.ModuleDict({'accuracy': Accuracy(),
-                                   'auroc': AUROC(num_classes=self.n_classes),
-                                   'precision': Precision(),
-                                   'recall': Recall(),
-                                   'f1score': F1()})
+                                  'auroc': AUROC(num_classes=self.n_classes),
+                                  'precision': Precision(),
+                                  'recall': Recall(),
+                                  'f1score': F1(),
+                                  'confusion_matrix': ConfusionMatrix(num_classes=self.n_classes+1)})
```

Comment:
Seeing this here: It would be good to add documentation for n_classes beyond what you have now. For two classes "0" and "1", n_classes should be set to 1, correct?

Reply:
Thanks, added to the docstring